ffcx/codegeneration/coordinate_mapping.py

```diff
@@ -447,13 +445,10 @@ def _compute_reference_coordinates_newton(L, ir, output_all=False):
     Xm = L.Symbol("%s_midpoint" % cellname)
 
     # Variables for stopping criteria
-    # TODO: Check if these are good convergence criteria,
-    #       e.g. is epsilon=1e-6 and iterations=degree sufficient?
-    max_iter = L.LiteralInt(degree)
-    epsilon = L.LiteralFloat(1e-6)
-    # TODO: Could also easily make criteria input if desired
-    # max_iter = L.Symbol("iterations")
-    # epsilon = L.Symbol("epsilon")
+    # FIXME: return an error code (e.g. -1) on non-convergence
+    # Max ten iterations seems plenty, epsilon^2=1e-12
+    max_iter = L.LiteralInt(10)
+    epsilon2 = L.LiteralFloat(1e-12)
```

Comment:
Ok for a quick-fix, but in future we should have a way of controlling this. Maybe passing these as arguments to `compute_reference_coordinates`?

Reply:
We need at least an error code before merging.